src/invoices/InvoiceLineForm/InvoiceLineForm.js

```diff
@@ -123,10 +121,12 @@ const InvoiceLineForm = ({
   } = initialValues;
   const totalAmount = calculateTotalAmount(formValues, invoice.currency);
   const isEditPostApproval = IS_EDIT_POST_APPROVAL(id, invoiceLineStatus);
+  const initialAccountNumber = useRef(accountNumber);
```

Comment:
Why do you need `useRef` to get the initial value of the account number? Form initial values should not change during form re-renders.